debian/control

```diff
@@ -0,0 +1,30 @@
+Source: amazon-ecr-credential-helper
+Section: misc
+Priority: optional
+Maintainer: Samuel Karp <skarp@amazon.com>
+Build-Depends:
+ debhelper (>= 8.0.0),
+ dh-golang (>= 1.1),
+ golang-go (>= 1.7),
+ git,
+ golang-github-aws-aws-sdk-go-dev,
+ golang-github-docker-docker-credential-helpers-dev,
+ golang-github-mitchellh-go-homedir-dev,
+ golang-github-sirupsen-logrus-dev,
+ golang-github-golang-mock-dev,
+ golang-github-pkg-errors-dev
+Standards-Version: 4.2.1
+Homepage: https://github.com/awslabs/amazon-ecr-credential-helper
+Vcs-Browser: https://github.com/awslabs/amazon-ecr-credential-helper
+Vcs-Git: https://github.com/awslabs/amazon-ecr-credential-helper
+XS-Go-Import-Path: github.com/awslabs/amazon-ecr-credential-helper
+
+Package: amazon-ecr-credential-helper
+Architecture: any
+Depends:
+ ${misc:Depends},
+ ${shlibs:Depends}
+Built-Using: ${misc:Built-Using}
+Description: Amazon ECR Credential Helper for Docker
+ A Docker credential helper to automatically manage credentials for Amazon ECR
```

Comment:
It's probably a good idea to extend the long description somewhat. If this package shows up in some search results, how will I know that it solves the problem I'm having? What does it mean to "automatically manage credentials"?

Reply:
Done.